config.go

```diff
@@ -43,10 +50,17 @@ func NewConfig(appName string) Config {
 // WithConfigEnvVar looks for the environment variable: <APP_NAME>_CONFIG as a way to specify a config file
 // This will be overridden by a command-line flag
 func (c Config) WithConfigEnvVar() Config {
-	c.File = os.Getenv(envVar(c.AppName, "CONFIG"))
+	c.Files = strings.Split(os.Getenv(envVar(c.AppName, "CONFIG")), ",")
 	return c
 }
 
 func (c *Config) AddFlags(flags FlagSet) {
-	flags.StringVarP(&c.File, "config", "c", fmt.Sprintf("%s configuration file", c.AppName))
+	if c.ConfigureMultipleFiles {
+		flags.StringArrayVarP(&c.Files, "config", "c", fmt.Sprintf("%s configuration file", c.AppName))
+	} else {
+		flags.StringVarP(&c.File, "config", "c", fmt.Sprintf("%s configuration file", c.AppName))
+	}
+	if c.ProfileKey != "" {
+		flags.StringArrayVarP(&c.Profiles, "config-profile", "", "configuration profiles to use")
```

Comment:
Naming suggestions welcome; this could also be configurable and/or match the ProfileKey somewhat

Reply:
`--profile` is my vote